api/tests/opentrons/protocol_engine/execution/test_equipment_handler.py

```diff
@@ -165,6 +166,44 @@ async def test_load_labware(
     )
 
 
+async def test_load_labware_off_deck(
+    decoy: Decoy,
+    model_utils: ModelUtils,
+    state_store: StateStore,
+    labware_data_provider: LabwareDataProvider,
+    minimal_labware_def: LabwareDefinition,
+    subject: EquipmentHandler,
+) -> None:
+    """It should load labware definition and offset data and generate an ID."""
+    decoy.when(model_utils.generate_id()).then_return("unique-id")
+
+    decoy.when(state_store.labware.get_definition_by_uri(matchers.IsA(str))).then_raise(
+        errors.LabwareDefinitionDoesNotExistError("oh no")
+    )
+
+    decoy.when(
+        await labware_data_provider.get_labware_definition(
+            load_name="load-name",
+            namespace="opentrons-test",
+            version=1,
+        )
+    ).then_return(minimal_labware_def)
```

Comment:
If it helps, we can simplify this test a little bit by taking an easier path through the production code:

```suggestion
    decoy.when(state_store.labware.get_definition_by_uri("opentrons-test/load-name/1").then_return(
        minimal_labware_def
    )
```

Reply:
nice